.github/workflows/chromatic.yaml

```
@@ -0,0 +1,17 @@
+
+on: push
+name: Test Deploy
+jobs:
+  build-test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
```

As discussed before:

I can not prove it b/c I don't know chromatic good enought to understand where the problems would show but I assume that this statement [here](https://github.com/chromaui/action#checkout-depth) is true no matter if we do the chromatic upload via their github action or npm script - if there is no git history from the `actions/checkout` action, then there is nothing to upload.

Consequently we must either use v1 (which seems to do all the history by default) _or_ add `fetch-depth: 0` and use v2 of `actions/checkout`. I'm in favor of the former.

```suggestion
      uses: actions/checkout@v1
```